wordpress-mobile/WordPress-Android
# Issue/use enum name when tracking comment filters

When tracking comment filter switches we are using localized filter label, instead of constant one.
This PR updates the tracking to use capitalized comment filter enum name and updates some of them to represent the actual filter labels.


To test:
- Put a breakpoint [here](https://github.com/wordpress-mobile/WordPress-Android/blob/ef033baa92ebc74fe41db4b0eefec5ce83798b46/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsActivity.kt#L106).
- Switch app language to non-English.
- Confirm that the `selected_filter` properties are `All`, `Pending`, `Unreplied`, `Approved`, `Spam`, and `Trashed`.

## Regression Notes
1. Potential unintended areas of impact
Hopefully none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
This is a legacy code, that is not easily testable. We are working on refactoring comments to remedy this.

PR submission checklist:
